### PR TITLE
Fixed sosreport failure on rhel9.0

### PR DESCRIPTION
**Pull request by Pavithra1602**
Fixed failure due to dist.release version lessthan 4 and profile option failure.

Signed-off-by: Pavithra <pavrampu@linux.vnet.ibm.com>

### CODE REVIEW

File: ras/sosreport.py [hunk 1]
```diff
@@ -56,8 +56,11 @@ def setUp(self):
...
                 "Package %s is missing and could not be installed" % sos_pkg)
-        if dist.name == "rhel" and dist.version > "7" and dist.release >= "4":
-            self.sos_cmd = "sos report"
+        if dist.name == "rhel":
+            if dist.version <= "7" and dist.release <= "4":
+                self.sos_cmd = "sosreport"
+            else:
+                self.sos_cmd = "sos report"
         else:
             self.sos_cmd = "sosreport"
```
Comment (sacsant) [inline ras/sosreport.py:66]: How about we simplify the above code as follows ?

```
self.sos_cmd = "sosreport"
if dist.name == "rhel":
             if dist.version > "7" and dist.release >= "4":
                 self.sos_cmd = "sos report"
```

File: ras/sosreport.py [hunk 2]
```diff
@@ -170,8 +173,8 @@ def test_others(self):
         self.run_cmd("%s --batch --tmp-dir=%s -p boot,memory" %
                      (self.sos_cmd, directory_name))
 
-        if "java" not in self.run_cmd_out("%s --batch --tmp-dir=%s -p webserver | "
-                                          "grep java" % (self.sos_cmd, directory_name)):
+        if "perl" not in self.run_cmd_out("%s --batch --tmp-dir=%s -p webserver | "
+                                          "grep perl" % (self.sos_cmd, directory_name)):
```
Comment (sacsant) [inline ras/sosreport.py:177]: This change seems unrelated. This does not seem to be related distro release and version.If still required can you include this in a separate commit?

**Comment by Pavithra1602** [comment]
Raising new PR.
